src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/EnumConverter.cs

```diff
@@ -25,10 +25,14 @@ internal sealed class EnumConverter<T> : JsonConverter<T>
 
         private readonly ConcurrentDictionary<ulong, JsonEncodedText> _nameCache;
 
+        private readonly Lazy<ConcurrentDictionary<ulong, JsonEncodedText>> _dictionaryKeyPolicyCache = new Lazy<ConcurrentDictionary<ulong, JsonEncodedText>>();
```

Comment:
We shouldn't need `Lazy` here, just a nullable `ConcurrentDictionary` instantiated only when needed should be fine. I know using the parameterless ctor here means that there'll be runtime reflection done to create the dictionary. 

cc @steveharter what's the guidance on using Lazy in new framework code today?